yarn-project/barretenberg.js/src/crypto/grumpkin/index.ts

```diff
@@ -1,7 +1,7 @@
-import { BarretenbergWasm } from '../../wasm/index.js';
+import { WasmWrapper } from '@aztec/foundation/wasm';
 
 export class Grumpkin {
```

Comment:
Not the biggest fan of the Wrapper name, the intent was that the generic concept was called WasmModule

Reply:
Almost any utility class could be called a Wrapper, is my reasoning, but it represents what it wraps

Reply:
Agree, and WasmModule was also my first choice for a name, but there's already a lower level WasmModule in the repo.

Reply:
Yeah I created that one, it was meant to provide everything needed to just pass around without introducing base classes

Reply:
I suppose that's not quite right though with async call state